### ISSUE TITLE
Add man comand to show the documentation for the method

### ISSUE THREAD

**Issue by evshvarov** [intersystems-ru/webterminal#88]
Sometimes you don't know exactly what classmethod does.
it would be great to introduce man command to get the following:

USER> man ##class(Class.Name).Method()
/// it is very valuable method, it has the following params... bla-bla

So this command will show all the /// lines before the method in the class's code

**Comment by eduard93** [comment]
man class method
or 
man class:method

man class
to show class docs

**Comment by eduard93** [comment]
@ChaosGreyder: добавьте еще туда в вывод FormalSpec

**Comment by ZitRos** [comment]
I think it makes sense to create "full-featured autocomplete with suggestions" here.

Like the following: when you have some autocomplete variant (class, method, etc), or when you have the method name typed, WebTerminal will show a hint consisting of the comment/docs (if there are ones). Showing the formal spec is possible too! I will try to design this as best as possible.

Wow, so much feedback and things to do, thanks!

**Comment by evshvarov** [comment]
@ZitRos man should be a separate command, cause sometimes you need thorough documentation and formalspec info for the class, methods, params, and once you use the method it 100-500 times in a day you don't care any formal spec already.

**Comment by ZitRos** [comment]
@evshvarov, why not just to type the code and get hints? As for the man command you suggest, you need also to type the names. Why not to make it elegant? The hont will appear with little delay so it won't disturb the user from every-day purposes. The functionality you describe is covered by documentation as well.

**Comment by evshvarov** [comment]
Yes, it's a kind of a "call to documentation". How can you see the
documentation of a non-system class which cannot be shown in
docs.intersystems.com?
The man command can provide an answer for "I forgot the params of the
method and I forgot what exactly method does, can I see this info?"
And man can print a story of documentation placed in /// before the method
(it can be long).
Delayed hints can help, but what I dislike that I don't need this info
every time, and that I should wait for the delay.

**Comment by ZitRos** [comment]
> How can you see the
> documentation of a non-system class which cannot be shown in
> docs.intersystems.com?

Caché always has local documentation in it, isn't it?

I agree with the info you need to see each time. How about to add the little (i) icon when hovering over methods/classes in input? And once you press this button, the hint with documentation will appear. I will try to make the best implementation for this!

**Comment by evshvarov** [comment]
Caché always has local documentation in it, isn't it?
> Yes, but it needs the appropriate rights on this server. And usually, doc
namespace is closed on a production site.

> I agree with the info you need to see each time. How about to add the
little (i) icon when hovering over methods/classes in input? And once you
press this button, the hint with documentation will appear. I will try to
make the best implementation for this!

How about man command? I would prefer separate command if and when I want
some documentation.

**Comment by ZitRos** [comment]
OK. I will make this, but I will definitely think about making cool suggestions with formal spec, class/classMembers docs, etc.
